### PR TITLE
CentrifugeJS: Create a pool & Usage documentation

### PR DESCRIPTION
**Pull request by sophialittlejohn**
### Description

This pull request...

Closes #<issue-number>

### Approvals

- [ ] Dev
- [ ] Dev
- [ ] Designer
- [ ] Product

### CODE REVIEW

File: centrifuge-js/README.md [hunk 1]
```diff
@@ -1,53 +1,156 @@
...
-// example query
-const pools = await lastValueFrom(centrifuge.pools.getPools())
+IPFS gateway url for retrieving metadata.
 
-// example extrinsic
-const connectedCent = centrifuge.connect(address, signer)
-const mintNft = () => {
-  return connectedCent.nfts.mintNft([<collection-id>, <nft-id>, <owner>, <metadata>, <amount>])
-}
+#### `centrifugeSubqueryUrl`
```
Comment (onnovisser) [inline centrifuge-js/README.md:49]: `altairSubqueryUrl`

File: centrifuge-js/README.md [hunk 2]
```diff
@@ -1,53 +1,156 @@
...
+```
+
+Then the query could look like this
+
+```js
+import { lastValueFrom } from 'rxjs'
+
+// ...
+
+const pools = await lastValueFrom(cenrtifuge.pools.getPools()) // Pool[]
```
Comment (onnovisser) [inline centrifuge-js/README.md:124]: `firstValueFrom` for queries, otherwise the Promise will never resolve

**Review by onnovisser — COMMENTED** [review]
Really nice! Left just a couple small comments

File: centrifuge-js/README.md (hunk 1 above)
Comment (sophialittlejohn) [reply centrifuge-js/README.md:49]: changed the URL instead

**Review by onnovisser — APPROVED** [review]
💯 💯

**Pull request merged**